production/loki-mixin-compiled-ssd/dashboards/loki-logs.json

```diff
@@ -63,7 +63,7 @@
             "steppedLine": false,
             "targets": [
                {
-                  "expr": "sum(go_goroutines{cluster=\"$cluster\", namespace=\"$namespace\", pod=~\"$deployment.*\", pod=~\"$pod\"})",
+                  "expr": "sum(go_goroutines{cluster=~\"$cluster\", namespace=\"$namespace\", pod=~\"$deployment.*\", pod=~\"$pod\"})",
```

Comment:
Replacing all `cluster=\"$cluster\"` with `cluster=~\"$cluster\"` to be the same on all clusters